AlpsSDK/ModelExtensions.swift

```diff
@@ -53,11 +52,9 @@ extension Device: Hashable {
     }
 }
 
-extension Publication {
-
-    internal convenience init(deviceId: String? = nil, topic: String, range: Double, duration: Double, properties: [String: String]) {
+public extension Publication {
+    public convenience init(deviceId: String? = nil, topic: String, range: Double, duration: Double, properties: [String: String]) {
```

Comment:
If there is no deviceId, it shouldn't be able to init

Reply:
True. Fixing now.